base_comment_template/migrations/16.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
```

Comment:
In this case, you have to check if the column already exists, as you may come from v15 with this patch already applied.